src/views/hardware_button.rs

```diff
@@ -4,6 +4,7 @@ use iced::{Color, Element, Length};
 use crate::styles::button_style::ButtonStyle;
 use crate::{Message, ToastMessage};
 
+const TEXT_WIDTH: u16= 16;
```

Comment:
Define a "default" text height once at the application level and have all modules use the same one, for consistency?